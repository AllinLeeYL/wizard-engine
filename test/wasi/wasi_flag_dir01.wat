;; A test case to confirm the behavior of directory related flags.
;; Therefore, wasm file itself is almost empty.

;; dir01: Error Case - wizeng should throw an error when specified directory does not exist
(module
  (func $get (import "wasi_snapshot_preview1" "clock_time_get") (param i32 i64 i32) (result i32))
  (func $main (export "main")
  )
)
