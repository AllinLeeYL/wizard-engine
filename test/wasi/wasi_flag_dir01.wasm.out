wasi_flag_dir01.wasm:0x0: preopen path is invalid: nonexistent_dir
